roles/git-server/tasks/main.yaml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Install Packages
+  action: "{{ ansible_pkg_mgr }} name={{ item }} state=present"
```

Comment:
Unless 1.9 support is needed here, I would suggest using the `package` module here.

If 1.9 support is needed, should probably define ansible_pkg_mgr in vars/main.yaml for the role.

Otherwise, lgtm.